DuckDuckGo/NetworkProtectionAccessController.swift

```diff
@@ -105,7 +105,9 @@ struct NetworkProtectionAccessController: NetworkProtectionAccess {
     }
 
     func revokeNetworkProtectionAccess() {
+#if os(macOS)
         try? networkProtectionKeychainTokenStore.deleteToken()
+#endif
```

Comment:
Any reason to keep the code inside the `os(macOS)` checks around given that this is iOS client-side code?